shogun-lib/src/main/java/de/terrestris/shogun/lib/model/jsonb/layer/DefaultLayerSourceConfig.java

```diff
@@ -75,5 +76,11 @@ public class DefaultLayerSourceConfig implements LayerSourceConfig {
         example = "© by terrestris GmbH & Co. KG"
     )
     private String attribution;
+
+    @Schema(
+        description = "Request parameters to be passed to the service when querying a layer.",
+        example = "requestParams: {transparent: true}"
```

Comment:
Is this working in the Swagger API/Docs as expected? I would expect something like `{"transparent": true}`.